bootstrap/src/basis/executor.c

```diff
@@ -0,0 +1,23 @@
+#include <threads.h>
+
+#include "common.h"
+#include "ioring.h"
+
+#include "executor.h"
+
+thread_local hm_executor_t executor = {0};
```

Comment:
Yeah. I had to dig into this one a bit because I'd originally written the `{}` as `{0}` and it bit me hard. This is a shorthand for zeroing the first field. All remaining fields take the same value.